Dockerfile.operator

```diff
@@ -37,12 +37,17 @@ RUN exportOrFail() { echo $1; if [[ $1 == *= ]]; then echo "Error: empty variabl
               -X ${VERSION_PKG}.autoInstrumentationApacheHttpd=${AUTO_INSTRUMENTATION_APACHE_HTTPD_VERSION} \
               -X ${VERSION_PKG}.autoInstrumentationNginx=${AUTO_INSTRUMENTATION_NGINX_VERSION}"
 
-FROM registry.redhat.io/ubi8/ubi-minimal:latest@sha256:33161cf5ec11ea13bfe60cad64f56a3aa4d893852e8ec44b2fd2a6b40cc38539
+
+FROM registry.redhat.io/ubi8/ubi:latest as ubi-micro-additional-packages
+RUN mkdir -p /mnt/rootfs
+RUN dnf install --installroot /mnt/rootfs --releasever 8 --setopt install_weak_deps=false --nodocs -y coreutils-single glibc-minimal-langpack openssl; dnf clean all
+RUN rm -rf /mnt/rootfs/var/cache/*
+
+
+FROM registry.redhat.io/ubi8/ubi-micro:latest@sha256:eae27ba458e682d6d830f6c77c9e3a4c33cf1718461397b741e674d9d37450f3
 WORKDIR /
 
-RUN microdnf update -y && rm -rf /var/cache/yum && \
-    microdnf install openssl -y && \
-    microdnf clean all
+COPY --from=ubi-micro-additional-packages /mnt/rootfs/ /
```

Comment:
```Dockerfile
FROM registry.redhat.io/ubi8/ubi-micro:latest@sha256:eae27ba458e682d6d830f6c77c9e3a4c33cf1718461397b741e674d9d37450f3 AS target-base

FROM registry.redhat.io/ubi8/ubi:latest as ubi-micro-additional-packages
COPY --from=target-base / /mnt/rootfs
RUN dnf install --installroot /mnt/rootfs --releasever 8 --setopt install_weak_deps=false --nodocs -y coreutils-single glibc-minimal-langpack openssl; dnf clean all
RUN rm -rf /mnt/rootfs/var/cache/*


FROM scratch

COPY --from=ubi-micro-additional-packages /mnt/rootfs/ /
WORKDIR /
```

Reply:
We are using hermetic builds and `from scratch` failed on 
```
[2/3] STEP 4/4: RUN . /cachi2/cachi2.env &&     rm -rf /mnt/rootfs/var/cache/*
[3/3] STEP 1/11: FROM scratch
[3/3] STEP 2/11: WORKDIR /
[3/3] STEP 3/11: COPY --from=ubi-micro-additional-packages /mnt/rootfs/ /
[3/3] STEP 4/11: RUN . /cachi2/cachi2.env &&     mkdir /licenses
process failed to start with error: fork/exec /bin/sh: no such file or directory
process exited with error: exec: not started
subprocess exited with status 1
Error: building at STEP "RUN . /cachi2/cachi2.env &&     mkdir /licenses": exit status 1
```